lib/Core/ErrorState.h

```diff
@@ -41,6 +41,18 @@ class ErrorState {
 
   std::map<uintptr_t, ref<Expr> > storedError;
 
+  bool isInStoredError(ref<Expr> address) const {
```

Comment:
@Himeshi I have no strong preference, but perhaps `hasStoredError` might be a better name for this member function.